packages/editor/src/bindings/post-meta.js

```diff
@@ -25,4 +25,48 @@ export default {
 			.getEditedEntityRecord( 'postType', postType, context.postId )
 			.meta?.[ args.key ];
 	},
+	setValue( { registry, context, args, value } ) {
+		const postType = context.postType
+			? context.postType
+			: registry.select( editorStore ).getCurrentPostType();
+		registry
+			.dispatch( coreDataStore )
+			.editEntityRecord( 'postType', postType, context.postId, {
+				meta: {
+					[ args.key ]: value,
+				},
+			} );
+	},
+	lockAttributesEditing( { select, context = {}, args } ) {
```

Comment:
It seems we will need these arguments for the `lockAttributesEditing` callback as well.

I don't know if we should pass the registry like in the other functions or the `select` is enough.

Any thoughts?

Reply:
I'm usually in favor on keeping consistency with other functions.